net-im/skypeforlinux/skypeforlinux-1.9.0.2.ebuild

```diff
@@ -93,11 +90,7 @@ src_install() {
 #	insinto /etc/dbus-1/system.d
 #	doins ${PN}.conf
 
-	dodoc usr/share/doc/skypeforlinux/* usr/share/skypeforlinux/*.html
-	dodoc usr/share/skypeforlinux/*.txt usr/share/skypeforlinux/LICENSE
-
-	# create compat symlink
-	dosym ${P} /usr/share/doc/skypeforlinux
+	dodoc usr/share/doc/skypeforlinux/*
```

Comment:
Also, I don't like the idea of calling `dodoc`, `doicon` post `into`. `into` doesn't seem to be consistently respected, and this may mean stuff might end up getting fixed to use it.